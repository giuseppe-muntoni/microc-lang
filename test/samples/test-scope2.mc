void foo() {
    print_endline("I'm foo");
}

void main() {
    foo();

    {
        int foo;
        foo = 5;
        print(foo);
    }

}