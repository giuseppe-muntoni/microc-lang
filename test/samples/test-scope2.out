I'm foo
5
